RELEASE.md

```diff
@@ -0,0 +1,53 @@
+Release
+=======
+
+Although not very tricky, the Ember Data release process does have a 
+lot of manual steps. The following steps navigate a through
+some of the release gotchas and will hopefully result in a successful
+release.
+
+STEPS:
+------
+
+* generate changelog (`PRIOR_VERSION=v2.0.0 HEAD=release ./bin/changelog`)
+* prepend changelog output to `CHANGELOG.md`
+* edit changelog output to be as user-friendly as possible (drop [INTERNAL] changes, non-code changes, etc.)
+* Bump version in package.json
+* Git tag version
+* Do a production build. `rm -rf node_modules bower_components; npm install; bower install; npm run production`
+* Publish to Bower
+  * Commit built globals code to the https://github.com/components/ember-data repo
+  * `cp dist/globals/* ../components-ember-data/`
+* Publish Rubygems 
+  * `gem build ember-data-source.gemspec` 
+  * `gem push ember-data-source-2.4.0.beta.1.gem`
+* Publish to NPM 
+  * `npm publish` or `npm publish --tag beta` 
+  * (If publishing a 1.13 release update latest to be 2.x `npm dist-tag add ember-data@2.1.0 latest`)
```

Comment:
Maybe there is a better way I could phrase this. This isn't intended to be a step that is done every time but only when doing `npm publish` for a 1.13.x release. Because then the npm site will say the latest release is 1.13.x instead of 2.x.x.